lib/web/JsonpMainTemplatePlugin.js

```diff
@@ -168,12 +169,7 @@ class JsonpMainTemplatePlugin {
 					),
 					"}",
 					"script.src = jsonpScriptSrc(chunkId);",
-					"var timeout = setTimeout(function(){",
-					Template.indent([
-						"onScriptComplete({ type: 'timeout', target: script });"
-					]),
-					`}, ${chunkLoadTimeout});`,
-					"function onScriptComplete(event) {",
+					"onScriptComplete = function (event) {",
```

Comment:
`function onScriptComplete(event) {` was fine

Reply:
As an inner declaration, it was not fine to some extends.

Reply:
https://eslint.org/docs/rules/no-inner-declarations